src/front/common/Icons.tsx

```diff
@@ -0,0 +1,40 @@
+//
+// LESERKRITIKK v2 (aka Reader Critics)
+// Copyright (C) 2017 DB Medialab/Aller Media AS, Oslo, Norway
+// https://github.com/dbmedialab/reader-critics/
+//
+// This program is free software: you can redistribute it and/or modify it under
+// the terms of the GNU General Public License as published by the Free Software
+// Foundation, either version 3 of the License, or (at your option) any later
+// version.
+//
+// This program is distributed in the hope that it will be useful, but WITHOUT
+// ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+// FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License along with
+// this program. If not, see <http://www.gnu.org/licenses/>.
+
+import * as React from 'react';
+
+export interface IIcon {
+	color?: string;
+	width?: string;
+	height?: string;
+}
+
+export const AlertIcon : React.StatelessComponent <IIcon> = (props : IIcon) => {
+	const color = props.color || '#f44336',
+		width = props.width || '56px',
+		height = props.height || '56px';
+	// tslint:disable-next-line
```

Comment:
Are you sure about that?

Reply:
Yes, the line is too big and can not be broken by lines in some places to prevent breaking the path code. In an example before it was implemented too